docs/docs/cli.md

```diff
@@ -482,6 +482,48 @@ associated with a specific project.
 
 See [Managing environments](/docs/managing-environments.md) for more information about these commands.
 
+## bundle
+
+The `bundle` command regroups sub commands to bundle the current project
+and its dependencies into various formats. These commands are particularly useful to deploy
+Poetry-managed application.
+
+### bundle venv
+
+The `bundle venv` command bundles the project and its dependencies into a virtual environment.
+
+The following command
+
+```bash
+poetry bundle venv /path/to/environment
+```
+
+will bundle the project in the `/path/to/environment` directory by creating the virtual environment,
+installing the dependencies and the current project inside it. If the directory does not exist
+it will be created automatically.
+
+By default, the command uses the current Python executable to build the virtual environment.
+If you want to use a different one, you can specify it with the `--python/-p` option:
+
+```bash
+poetry bundle venv /path/to/environment --python /full/path/to/python
+poetry bundle venv /path/to/environment -p python3.8
+poetry bundle venv /path/to/environment -p 3.8
+```
+
+!!!note
+
+    If the virtual environment already exists, two things can happen:
+
+    - **The python version of the virtual environment is the same as the main one**: the dependencies will be synced (updated or removed).
+    - **The python version of the virtual environment is different**: the virtual environment will be recreated from scratched.
```

Comment:
scratch ?

Reply:
> I haven't looked thru the implementation in detail - but is the goal to achieve something like what shiv is doing ? I've been using poetry build + shiv to produce a deployable artifact. Great if this can all be done thru poetry. I guess the key difference is that Shiv is also packing things into an executable .zip ( .pyz ), but otherwise the packaging of dependencies looks pretty similar.

@Imaclean74 any chance you can provide more details on this?